doc/nrf/ug_tfm.rst

```diff
@@ -101,12 +101,24 @@ Logging
 TF-M employs two UART interfaces for logging: one for the secure part (MCUboot and TF-M), and one for the non-secure application.
 The logs arrive on different COM ports on the host PC.
 
-.. note::
-   * On the nRF5340 DK v1.0.0, you must connect specific wires on the kit to receive secure logs on the host PC.
-     Specifically, wire the pins **P0.25** and **P0.26** of the **P2** connector respectively to **RxD** and **TxD**  of the **P24** connector.
-     See :ref:`logging_cpunet` on the Working with nRF5340 DK page for more information.
-   * On the nRF5340 DK v2.0.0, there is one fewer COM port than on v1.0.0, so the secure and non-secure UART peripheral must be wired to the same pins.
-     Specifically, wire the pins **P0.25** and **P0.26** to **P0.20** and **P0.22**, respectively.
+
+VCom Ports nRF5340 DK
+=====================
+
+* v1.0.0: You must connect specific wires on the kit to receive secure logs on the host PC.
+  Specifically, wire the pins **P0.25** and **P0.26** of the **P2** connector respectively to **RxD** and **TxD**  of the **P24** connector.
+  See :ref:`logging_cpunet` on the Working with nRF5340 DK page for more information.
+* v2.0.0: There are only two virtual COM ports available on that version. 
+  By default one will be used by the non-secure UART peripheral of of the application and the other one by the UART1 peripheral from the network core.
+  There are several options to get UART output from the secure TF-M.
+1. Disable the output for the network core.
```

Comment:
Also, does this have to be a numbered list? Are they just the three options? Because now they look more like steps that need to be followed in this order.